src/server/journal/journal.cc

```diff
@@ -40,10 +40,6 @@ void Journal::StartInThread() {
 error_code Journal::Close() {
   VLOG(1) << "Journal::Close";
 
-  if (!journal_slice.IsOpen()) {
```

Comment:
redundant after we call `journal_slice.Init` once. Won't make any difference either if it's not open